py-rattler/rattler/version/version.py

```diff
@@ -54,6 +65,178 @@ def bump(self) -> Version:
         """
         return Version._from_py_version(self._version.bump())
 
+    @property
+    def has_local(self) -> bool:
+        """
+        Returns true if this version has a local segment defined.
+        The local part of a version is the part behind the (optional) `+`.
+
+        Examples
+        --------
+        >>> v = Version('1.0+3.2-alpha0')
+        >>> v.has_local
+        True
+        """
+        return self._version.has_local()
+
+    def segments(self) -> List[List[str]]:
+        """
+        Returns a list of segments of the version. It does not contain
+        the local segment of the version.
```

Comment:
Can you add a reference to `local_segments` to indicate that people can use that too?